src/commands/search_tx_optimizations.rs

```diff
@@ -65,29 +65,22 @@ impl SearchTxOptimizationsCommand {
   async fn drop_optimizations(&self, pool: &PgPool) -> Result<()> {
     println!("Dropping search transaction optimizations from Archive Database...");
 
-    // Load and execute the SQL from the file
-    let sql = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/sql/migrations/drop_search_tx_optimizations.sql"));
-    self.execute_sql_file(pool, sql, ";").await?;
+    MIGRATOR.undo(pool, 0).await?;
 
     println!("Optimizations dropped successfully.");
     Ok(())
   }
 
   async fn check_if_optimizations_applied(&self, pool: &PgPool) -> Result<bool> {
-    // select if table exists
-    let result = sqlx::query(
-      "SELECT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'user_commands_aggregated')",
-    )
-    .fetch_one(pool)
-    .await?;
-    Ok(result.get(0))
-  }
+    // select the latest migration version in the DB
```

Comment:
In case the migrations have not been applied yet (like via `sqlx migrate run --source sql/migrations`), we get something like:

```
$ target/debug/mina-mesh search-tx-optimizations --apply
Error: error returned from database: relation "_sqlx_migrations" does not exist

Caused by:
    relation "_sqlx_migrations" does not exist
```

I think the solution could be checking if the table `_sql_migrations` exists here at the beginning  of `check_if_optimizations_applied`, like:

```suggestion
    // Check if the migrations table exists
    let table_exists: Option<String> =
      sqlx::query_scalar("SELECT to_regclass('_sqlx_migrations')::text").fetch_one(pool).await?;

    if table_exists.is_none() {
      // The table doesn't exist so the optimizations have not been applied
      return Ok(false);
    }

    // select the latest migration version in the DB
```

Reply:
Just a side note, I did not include the table check because the max version query worked for me, even when the table did not exist. I'm guessing this could be some Postgres setting to behave differently when tables do not exist, so it's best to include this table check to avoid these situations.